spec/widgets/category/options-view.spec.js

```diff
@@ -109,17 +105,8 @@ describe('widgets/category/options-view', function () {
     it('should render filter buttons if widget is neither locked nor search enabled', function () {
       spyOn(this.widgetModel, 'isLocked').and.returnValue(false);
       spyOn(this.widgetModel, 'isSearchEnabled').and.returnValue(false);
-      this.dataviewModel.filter.accept('Hey');
-      this.dataviewModel.sync = function (method, model, options) {
-        options.success({
-          'categories': [
-            {category: 'Hey'},
-            {category: 'Buddy'}
-          ]
-        });
-      };
-      this.dataviewModel.fetch();
-      this.view.render();
+      this.dataviewModel._data.reset([{ name: 'hey' }, { name: 'buddy' }, { name: 'neno' }]);
```

Comment:
Would something like `spyOn(this.dataviewModel, 'getData').and.returnValue([...])` work here? (just in case the dataview model changes internally and this breaks)